### PR TITLE
Update Rails to 5.2.4.5

### PR DESCRIPTION
**Pull request by rsanheim**
Fixes one security vulnerabilities in PostgreSQL / ActiveRecord:

The changeling is very small: https://github.com/rails/rails/compare/v5.2.4.4...v5.2.4.5

See
https://weblog.rubyonrails.org/2021/2/10/Rails-5-2-4-5-6-0-3-5-and-6-1-2-1-have-been-released/
for the details

**Story card:** [ch2823](https://app.clubhouse.io/simpledotorg/story/2823/update-rails-to-latest-secret-5-2-x)

### CODE REVIEW

**Review by kpethtel — APPROVED** [review]
Good to see us keeping current.

File: .semaphore/semaphore.yml [hunk 1]
```diff
@@ -32,10 +32,10 @@ blocks:
             - 'wget -O /tmp/libpng12.deb http://mirrors.kernel.org/ubuntu/pool/main/libp/libpng/libpng12-0_1.2.54-1ubuntu1_amd64.deb'
             - sudo dpkg -i /tmp/libpng12.deb
             - gem install semaphore_test_boosters
-            - cache restore
+            # - cache restore
```
Comment (harimohanraj89) [inline .semaphore/semaphore.yml:35]: Was this for testing? If so, we can undo this before we ship.

Comment (rsanheim) [reply .semaphore/semaphore.yml]: yup semaphore + bootsnap was giving me issues...I re-enabled it.

Comment (rsanheim) [reply .semaphore/semaphore.yml]: > yup semaphore + bootsnap was giving me issues...I re-enabled it.

semaphore cache is broken w/ this rails upgrade, not sure what is going on.  😢

**Pull request merged**